doc/devices/numpy_wavefunction.rst

```diff
@@ -0,0 +1,77 @@
+The Numpy-Wavefunction device
+=============================
+
+The ``forest.numpy_wavefunction`` device provides an interface between PennyLane
+and the pyQuil `NumPy wavefunction simulator <http://docs.rigetti.com/en/stable/wavefunction_simulator.html>`_.
+
+Because the NumPy wavefunction simulator allows access and manipulation of the underlying
+quantum state vector, ``forest.numpy_wavefunction`` is able to support the full
+suite of PennyLane and Quil quantum operations and observables.
+
+
+In addition, it is generally faster than running equivalent simulations on the QVM, as the
+final state can be inspected and the expectation value calculated analytically, rather than by sampling measurements.
```

Comment:
Hmm, not quite sure I agree here. From my experience, the main reason for slower execution with the QVM is that a local Lisp server is used that accepts/sends queries. Furthermore, the `forest.numpy_wavefunction` device could also be run in `analytic=False` mode.

Reply:
Deleted that statement.